src/main/java/com/minecolonies/coremod/colony/jobs/JobSwineHerder.java

```diff
@@ -0,0 +1,55 @@
+package com.minecolonies.coremod.colony.jobs;
+
+import com.minecolonies.coremod.client.render.RenderBipedCitizen;
+import com.minecolonies.coremod.colony.CitizenData;
+import com.minecolonies.coremod.entity.ai.basic.AbstractAISkeleton;
+import com.minecolonies.coremod.entity.ai.citizen.herders.EntityAIWorkSwineHerder;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * The SwineHerder job
+ */
+public class JobSwineHerder extends AbstractJob
+{
+    /**
+     * Instantiates the placeholder job.
+     *
+     * @param entity the entity.
+     */
+    public JobSwineHerder(final CitizenData entity)
+    {
+        super(entity);
+    }
+
+    @NotNull
+    @Override
+    public String getName()
+    {
+        return "com.minecolonies.coremod.job.SwineHerder";
```

Comment:
This can't cause issues with other languages? Do we use this anywhere where it might affect something?

Reply:
lemme take a look, i wouldn't have done it if i hadn't seen it elsewhere, i think

Reply:
it's used in all our other jobs:
![screen shot 2018-01-04 at 8 46 58 pm](https://user-images.githubusercontent.com/28527281/34560149-79d00b9e-f190-11e7-95e7-c58f3f3cd503.png)
![screen shot 2018-01-04 at 8 47 07 pm](https://user-images.githubusercontent.com/28527281/34560150-7a280eac-f190-11e7-9000-79d28ee3fe1d.png)f

Reply:
okay good. I wasn't sure if we use it everywhere for the job name.

Reply:
forgot to add 'em to lang tho, just fixed that